app/models/cohort.rb

```diff
@@ -35,6 +35,17 @@ def class_dates_until(last_date)
     (start_date..last_date).select { |date| !date.saturday? && !date.sunday? }
   end
 
+  def internships_sorted_by_interest(current_student)
+    self.internships.sort_by do |internship|
```

Comment:
you can take off the `self.` here; that's the [generally preferred style](https://github.com/bbatsov/ruby-style-guide#no-self-unless-required).